components/prize_pool/commons/prize_pool_legacy.lua

```diff
@@ -282,7 +283,7 @@ function LegacyPrizePool.mapOpponents(slot, newData, mergeSlots)
 			team = slot['team' .. opponentIndex],
 			lastvs = {
 				slot['lastvs' .. opponentIndex],
-				link = slot['lastvs' .. opponentIndex .. 'link'] or slot['lastvspage' .. opponentIndex],
+				link = slot['lastvs' .. opponentIndex .. 'link'] or slot['lastvs' .. opponentIndex .. 'page'],
```

Comment:
imo not change the alias but set another one
`slot['lastvspage' .. opponentIndex]` is used by some wikis apparently (as i was aked to add explicitly that)